.github/ISSUE_TEMPLATE/feature-request.md

```diff
@@ -0,0 +1,20 @@
+---
+name: Feature Request
+about: Suggest an idea for this project
+title: 'User Feature Request: *{insert feature request here}*'
+labels: user-visible
+assignees: ''
+
+---
+
+**Is your feature request related to a problem? Please describe.**
+A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
+
+**Describe the solution you'd like**
+A clear and concise description of what you want to happen.
```

Comment:
A clear and concise description of the desired outcome.

Do you prefer this formulation?

Reply:
Yes

Reply:
Done